0800-appendix-57793-vallee.Rmd

```diff
@@ -44,30 +56,57 @@ fpr::fpr_table_wshd_sum(site_id = my_site) %>%
 
 ```
 
-
 <br>
 
-Vallee Lake is `r fpr::fpr_my_bcfishpass(site = my_site, col_pull = total_lakereservoir_ha)`ha in size and modelling indicates an additional `r fpr::fpr_my_bcfishpass(col_pull = total_wetland_ha)`ha of wetland upstream of the highway.  Large lakes within watersheds can throttle high freshet and storm flows, increasing stability downstream for egg incubation and juvenile rearing. 
+Vallee Lake is
+`r fpr::fpr_my_bcfishpass(site = my_site, col_pull = total_lakereservoir_ha)`ha
+in size and modelling indicates an additional
+`r fpr::fpr_my_bcfishpass(col_pull = total_wetland_ha)`ha of wetland
+upstream of the highway. Large lakes within watersheds can throttle high
+freshet and storm flows, increasing stability downstream for egg
+incubation and juvenile rearing.
 
 <br>
 
-PSCIS stream crossing `r as.character(my_site)` was ranked as a high priority for follow up by the Canadian Wildlife Federation due to significant amounts of habitat modeled upstream. @irvine2018AnalysisPriority ranked the site as a moderate priority for follow up after review of PSCIS data/photos because the site appeared to have low flows, be partially backwatered and have no outlet drop indicating it was likely passable to most species and life stages at the time the initial fish passage assessment was conducted (2012).  A summary of habitat modelling outputs is presented in Table \@ref(tab:tab-culvert-bcfp-57793). A map of the watershed is provided in map attachment [`r fpr::fpr_my_bcfishpass(col_pull = dbm_mof_50k_grid)`](`r fpr::fpr_my_mapsheet()`).  
+Further downstream, GIS modelling indicates there is one crossing (modelled crossing ID 1802482) before the confluence of the Bulkley River. However, local knowledge from a landowner revealed that the crossing was no longer present. Upstream of crossing 57793, review of GIS indicates there are three crossings located on the mainstem of Vallee Creek, along with several modelled crossings on tributaries that are likely very small. PSCIS crossing 57801 is located past Vallee Lake on Highway 16 and was listed as a barrier at the time of survey by @gollner2013 in 2012. 
```

Comment:
crossing is modelled on a transmission line and landowner reported it does not exist.

Need to list the three crossings on the mainstem, key details of location (before the lake) and how far from the subject crossing to the first one.  

Are those modelled crossings not on the mainstem or are there some that are PSCIS - just confirming....

Reply:
good to include that detail (name of modelled crossing and name of road)

there are two lakes! They should be named in the text perhaps along with their sizes. Use Q to understand watershed and pass along key points to the reader.

Reply:
There is one lake before the modelled crossing that is very small (around 0.7ha) and doesn't even have a name in Q. Still worth mentioning? Or are you talking about Helen Lake that's on the other side of the highway?

Reply:
Helen. Its of significant in size, included in your lake area and if I remember correctly not mentioned yet in the text